scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -31,6 +31,55 @@ UBUNTU_VERSION=
 DISTRO=
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 INSTALL_DRIVER_VERSION='19.41.14441'
+AVAILABLE_DRIVERS=("19.41.14441" "20.35.17767")
+
+
+print_help()
+{
+    # Display Help
+    usage="Usage: $(basename "$0") [OPTIONS]...
+Download and installs the Intel® Graphics Compute Runtime for OpenCL™ Driver on Linux
+
+    Available options:
+    -y                      Replace the currently installed driver with the newer version.
+    -d, --install_driver    Manually select driver version to one of available to install drivers.
+                            Default value: $INSTALL_DRIVER_VERSION
+                            Available to install drivers: ${AVAILABLE_DRIVERS[*]}
+    -h, --help              display this help and exit" 
+    echo "$usage"
+}
+
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+    case $key in
+        -d|--install_driver)
+        user_chosen_driver="$2"
+        if [[ " ${AVAILABLE_DRIVERS[@]} " =~ " ${user_chosen_driver} " ]]; then
+            INSTALL_DRIVER_VERSION=$user_chosen_driver
+        else
+            echo "ERROR: unable to install the driver ${user_chosen_driver}."
+            echo "Available values: ${AVAILABLE_DRIVERS[*]}"
+            exit -1
+        fi
+        shift
+        shift
+    ;;
+        -y)
+        agreement="$2"
```

Comment:
Problem with `-y` option has not been fixed. Script must be run with `-y -y` option for it to work correctly. And I suppose arguments like `-y -d 20.35.17767` will not work at all.

Reply:
Fixed now in this PR and for release branch: https://github.com/openvinotoolkit/openvino/pull/3322